README.md

```diff
@@ -51,18 +51,42 @@ Build artifacts using maven commands as follows.
 
 #### Case that use embedded H2 as database
 
+The commands are different for XML-based configuration and Java-based configuration.
+
+** XML-based configuration **
```

Comment:
- [ ]  I think the "**" is for emphasis, but since there is a half-width space in between, it is judged as a normal string.

Reply:
Corrected the description error of markdown notation